cumulusci/tasks/salesforce/ActivateProcessBuilderProcesses.py

```diff
@@ -0,0 +1,36 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class ActivateProcessBuilderProcesses(BaseSalesforceApiTask):
+    """
+    Create a Salesforce Activation Process Builder which takes in a list as options and
+    runs flows only on those listed DeveloperName's.
+    """
+
+    def _run_task(self):
+        self.logger.info(
+            f'Allowing only rows with the following DeveloperNames {self.options.get("developer-names")}'
+        )
+        self.logger.info("Querying flow definitions...")
+        res = self.tooling.query(
+            "SELECT Id,ActiveVersion.VersionNumber,LatestVersion.VersionNumber,DeveloperName FROM FlowDefinition"
```

Comment:
Nit-pick: Let's add spaces between the fields in the SELECT.

Did you experiment at all with adding a `WHERE` clause here? The Tooling API is tetchy about how you can filter on certain fields, but if it will allow a filter on `DeveloperName` (and I think it will), we could add something like

`WHERE DeveloperName IN ({0})"`.format(",".join([f"'{n}'" for n in self.options.get("developer_names")]))`

Reply:
Yeah that looks right I can do this since it will probably make it faster.